go/pserver/client/c/test/test_train.py

```diff
@@ -31,6 +31,13 @@ def main():
 
     # create optimizer of new remote updater to pserver
     optimizer = paddle.optimizer.Momentum(momentum=0, learning_rate=1e-3)
+    optimizer.set_remote_optimizer_config(
```

Comment:
Sorry by V2 proto I meant `OptimizerConfig.proto`, V1 proto I meant `TrainerConfig.proto` and `ParameterConfig.proto`.

1. I meant V2 API, not the V2 parameter config (OptimizerConfig.proto) .
1. Just for clarity, I think the V2 API work with the old parameter server as well. Yes, OptimizerConfig.proto only works for the new pserver. (Sorry too much different V2 here...)

Yes agree on removing Python API only, for less confusion.